guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,355 @@
+---
+layout: default
+title: Theme Quality Assurance
+subhead: Prior to launching any ecommerce site, review this list and verify the site is up to Engine standards.
+has_children: false
+parent: Guides
+nav_order: 5
+permalink: /guides/theme-qa/
+---
+
+{% include page_header.html %}
+
+By following these guidelines, theme builders can ensure their customers receive the world’s best eCommerce experience on the Engine platform.
+
+[This document is open-source]({{ site.repo_url }}) and is built to be expanded. By continually revising this document, we will ensure that problems are never encountered more than once. If you encounter problems, please [open an issue]({{ site.repo_issues_url }}) or [make a contribution]({{ site.repo_url }}).
+
+Much of this test and QA process can be automated. However, there is great value in human experience, so we recommend a **real** human being verify all final quality tests before launch.
+
+---
+
+##### Table of Contents
+{:.no_toc}
+
+* An unordered list to be replaced with the ToC, excluding the "##### Table of Contents" from above
+{:.no_toc}
+{:toc levels=1}
+
+---
+
+## Testing Environments
+
+To maximize ecommerce conversion rates, verify these tests in the following browser environments:
+
+### Desktop
+{:.no_toc}
+
+* Windows
+  * Firefox 60+
+  * Chrome C66+
+  * IE 11+
+  * Edge 16+
+  * Opera 53+
+* Mac
+  * Safari 10+
+  * Chrome C66+
+  * Firefox 60+
+  * Opera 53+
+
+### Tablet
+{:.no_toc}
+
+* Android
+  * Current Galaxy
+  * Marketshare Winner
+* iOS
+  * Current iPad Pro
+  * Current iPad
+
+### Mobile
+{:.no_toc}
+
+* Android
+  * A currently-leading Samsung Galaxy phone
+  * A currently-leading Google Pixel phone
+* iOS
+  * A currently-leading large iPhone
+  * A currently-leading small iPhone
+
+*Note: the needs of every client are unique, so thoroughly research and test the environments used most by their target customers in addition to those above.*
+
+---
+
+## Theme Sections
+
+Verify each of the following stories are true:
+
+### Home Page/Overall
+{:.no_toc}
+
+* Home page should be width-responsive and should behave as expected at phone (landscape and portrait), tablet (landscape and portrait), and desktop resolutions and aspect ratios
+* Products visible on the Home page should link to product detail pages
+* Home page navigation should be present, usable, and consistent throughout the site
+* Client’s brand colors should be ubiquitous and consistently accurate throughout the site
+* Typography should be legible and styled in a typeface representative of the client’s brand
+* Meta data and other social and SEO structures should be present and well-formed
+* Tracking scripts should track views of the home page correctly
```

Comment:
Because you have capitalized "Home" in "Home page" up until this point, I think that this should be changed to mirror the capitalization of the other instances. 😄